src/data/librairies.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "name": "react"
+    },
+    {
+        "name": "preact"
+    },
+    {
+        "name": "svelte"
+    },
+    {
+        "name": "vue"
+    },
+    {
+        "name": "blank"
+    }
+]
```

Comment:
typo in the file name? I wonder if `src/data/librairies.json` should have been `src/data/libraries.json`? But then I see you used the term at several places throughout the code base, I guess you prefer French? 😄

Reply:
ahahah sorry for that, I'm french so I shuffled english and french, I fix that xDD

Reply:
pas de problème :)